manifests/kustomize/base/installs/multi-user/pipelines-profile-controller/sync.py

```diff
@@ -339,6 +340,51 @@ def sync(self, parent, children):
                         }
                     }
                 },
+                {
```

Comment:
Suggest adding a header so this doesn't look like part of the above artifact fetcher resources (I'm not particular about the wording, just don't want it to read that this is part of the artifact server)
```suggestion
                # Adds "Allow access to Kubeflow Pipelines" button in Notebook spawner UI
                {
```